apps/server/src/config.ts

```diff
@@ -22,45 +12,39 @@ const nonSensitiveSharedConfig = {
 const keysToPullFromEnv = [
   "PORT",
 
-  "HOSTURL",
-  "WebhookHostUrl",
-  "LogLevel",
+  "HOST_URL",
+  "WEBHOOK_HOST_URL",
+  "LOG_LEVEL",
 
   "UCPClientId",
   "UCPClientSecret",
```

Comment:
I can't remember--are we using these anymore?

Reply:
They're used for polling the institution service, but there is no institution service yet